app/models/manageiq/providers/openstack/cloud_manager.rb

```diff
@@ -177,4 +177,27 @@ def vm_delete_evm_snapshot(vm, image_id)
   def self.event_monitor_class
     ManageIQ::Providers::Openstack::CloudManager::EventCatcher
   end
+
+  #
+  # Statistics
+  #
+
+  def cinder_disk_usage
+    cinder_used = 0
+    cloud_volumes.each do |volume|
+      cinder_used += volume.size if volume.status != "error"
+    end
+    cloud_volume_snapshots.each do |volume|
+      cinder_used += volume.size if volume.status != "error"
+    end
+    cinder_used
```

Comment:
@Ladas That's good to know. I haven't looked at Ceph. We can adjust this once we understand how Ceph replication works.